components/raftstore/src/store/worker/pd.rs

```diff
@@ -889,19 +889,13 @@ where
         stats.set_capacity(capacity);
 
         let used_size = self.snap_mgr.get_total_snap_size().unwrap()
-            + store_info.engine.get_engine_used_size().expect("cf");
+            + store_info.engine.get_engine_used_size().expect("cf")
+            + disk::get_disk_reserved_space() / 5;
```

Comment:
The file may doesn't exist. `fstate` the file is better.

Reply:
Removed, to keep Pd store heartbeat used size as the original